try:
    print(x)
except Exception as e:
        print(str(e))
